### PR TITLE
Homework13: Marie Ambrazhei 

### PR DESCRIPTION
**Pull request by MarieAmbrazhei**

### CODE REVIEW

File: homework_13/homework_13_1.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    """This class define cards deck"""
+
+    def __init__(self, mast_list, number_list):
+        """This method is used to create cards deck"""
+        self.cards = []
+
+        for mast in mast_list:
+            for number in number_list:
+                self.cards.append(Card(mast, number))
+        self.cards.append(Card("Joker", None))
```
Comment (Stml89) [inline homework_13/homework_13_1.py:30]:
```suggestion
        self.cards.append(Card("red", "Joker"))
        self.cards.append(Card("black", "Joker"))
```

File: homework_13/homework_13_1.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    def shuffle(self):
+        """This method is used to shuffle cards deck"""
+        random.shuffle(self.cards)
+        return self.cards
+
+    def get(self, index):
+        """This method is used to get cards deck"""
+        if 0 <= index < len(self.cards):
+            return self.cards[index].number, self.cards[index].mast
```
Comment (Stml89) [inline homework_13/homework_13_1.py:41]:
```suggestion
            return self.cards[index]
```

File: homework_13/homework_13_1.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+CONTINUE_GAME = 'yes'
+
+while CONTINUE_GAME.lower() in ['y', 'yes']:
+    deck = CardsDeck(Card.mast_list, Card.number_list)
+    deck.shuffle()
+    card_number = int(input('Выберите карту из колоды в 56 карт: '))
+    card = deck.get(card_number)
+    print(f'You card is: {card[0]} {card[1]}')
```
Comment (Stml89) [inline homework_13/homework_13_1.py:53]: if you return object, so you can use it's attributes 

```suggestion
    print(f'You card is: {card.number} {card.mast}')
```

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework_13/homework_13_1.py (hunk 1 above)
Comment (MarieAmbrazhei) [reply homework_13/homework_13_1.py:30]: thanks, fixed

File: homework_13/homework_13_1.py (hunk 2 above)
Comment (MarieAmbrazhei) [reply homework_13/homework_13_1.py:41]: thanks, fixed

File: homework_13/homework_13_1.py (hunk 3 above)
Comment (MarieAmbrazhei) [reply homework_13/homework_13_1.py:53]: fixed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**